responsibleai_text/responsibleai_text/rai_text_insights/rai_text_insights.py

```diff
@@ -264,16 +264,15 @@ def _validate_model(self, model: Any, test: pd.DataFrame,
         """
         if not isinstance(target_column, list):
             target_column = [target_column]
-        # Pick one row from test data
-        small_test_data = test.iloc[0:1].drop(
+        # Use full test dataset for validation
+        test_data = test.drop(
             target_column, axis=1)
-        small_test_data = get_text_columns(small_test_data, text_column)
-        small_test_data = small_test_data.iloc[0]
+        test_data = get_text_columns(test_data, text_column)
         if task_type != ModelTask.QUESTION_ANSWERING:
-            small_test_data = small_test_data.tolist()
+            test_data = test_data.tolist()
         # Call the model
         try:
-            model.predict(small_test_data)
+            model.predict(test_data)
```

Comment:
For text this will be very slow - I don't suggest going down this route.  Text predictions are much slower than tabular.  Is there any reason why we need to do this just for validation?

Reply:
changed back to raising user error in error analysis manager predict call